### PR TITLE
add: GraphQL queries for Top users, posts, and tags

### PR DESCRIPTION
**Pull request by Dayitva**

### CODE REVIEW

File: socialite/server/graphql/resolvers/posts.js [hunk 1]
```diff
@@ -15,6 +15,74 @@ module.exports = {
...
+                    {'name' : 'Science', 'value' : 0},
+                    {'name' : 'Discrete Structures', 'value' : 0},
+                    {'name' : 'Speech Processing', 'value' : 0},
+                    {'name' : 'Computer Vision', 'value' : 0},
+                    {'name' : 'Internships', 'value' : 0},
+                    {'name' : 'Resume', 'value' : 0},
+                    {'name' : 'Computational Linguistics', 'value' : 0},
+                    {'name' : 'Algorithm Analysis & Design', 'value': 0},
+                    {'name' : 'NLP', 'value' : 0}
+                ]
```
Comment (prajneya) [inline socialite/server/graphql/resolvers/posts.js:61]: If we add more tags while scaling, we will have to add tags here too, right? Isn't there a way to fetch all tags and build this array automatically?

File: socialite/server/graphql/typeDefs.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ module.exports = gql`
     	createdAt: String!
         rating: Int!
         volatility: Float!
-        time_answered: Int!
+        times_answered: Int!
```
Comment (prajneya) [inline socialite/server/graphql/typeDefs.js:64]: This change does not alter the code flow at other files, right?

Comment (Dayitva) [reply socialite/server/graphql/typeDefs.js:64]: No, it does not. I checked.

File: socialite/server/graphql/resolvers/posts.js (hunk 1 above)
Comment (Dayitva) [reply socialite/server/graphql/resolvers/posts.js:61]: There might be. However, this is a first implementation and also has a close to n^3 running time. We can optimise this later.

**Pull request merged**